tools/alpine/Dockerfile

```diff
@@ -25,7 +27,10 @@ RUN cp /mirror/$(uname -m)/APKINDEX.unsigned.tar.gz /mirror/$(uname -m)/APKINDEX
 RUN abuild-sign /mirror/$(uname -m)/APKINDEX.tar.gz
 
 # fetch OVMF for qemu EFI boot (this is not added as a package)
-RUN apk add -X http://dl-cdn.alpinelinux.org/alpine/edge/community ovmf
+RUN mkdir -p /usr/share/ovmf && \ 
```

Comment:
This should probably be added the same way the wireguard tools were added a yesterday. The way this is currently added (and was in the past) does not record the version number in the `versions` file. I know it's not related directly to your PR, so maybe should be addressed separately as it also has implications elsewhere.